cola/widgets/about.py

```diff
@@ -36,7 +36,7 @@ class ExpandingTabBar(QtWidgets.QTabBar):
     """
 
     def tabSizeHint(self, tab_index):
-        width = self.parent().width() / max(1, self.count()) - 1
+        width = self.parent().width() // max(1, self.count()) - 1
```

Comment:
Thanks! I'll squash in a fixup to make this use `max(2, ...)` to remove any possibility of a divide-by-zero. That's pre-existing, though.